work/1 Table.sql

```diff
@@ -0,0 +1,18 @@
+use costumeDB
+go
+drop table if exists Costumes 
+go
+create table dbo.Costumes(
+CostumeId int not null identity primary key,
+CustomerName varchar (50) not null constraint ck_costumes_customername_cannot_be_blank check(customername <> ''),
+CostumesDesc varchar (20) not null constraint ck_costumes_CostumesBought_cannot_be_blank check (CostumesDesc <> ''),
```

Comment:
-- LL see you changed column name, great! fix the constraint name on CostumesDesc so that the name matches the new column name